src/scripts/ui.ts

```diff
@@ -382,8 +382,9 @@ export class ComfyUI {
       accept: '.json,image/png,.latent,.safetensors,image/webp,audio/flac',
       style: { display: 'none' },
       parent: document.body,
-      onchange: () => {
-        app.handleFile(fileInput.files[0])
+      onchange: async () => {
```

Comment:
Can you add a playwright testcase for this change? See https://github.com/Comfy-Org/ComfyUI_frontend/tree/main/browser_tests

Reply:
You can use https://github.com/Comfy-Org/ComfyUI_frontend/blob/2387a5e9bdb5fc30047c765b2788fecec465c318/browser_tests/ComfyPage.ts#L326-L331 for file upload.